src/environments/environment.ts

```diff
@@ -3,7 +3,8 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  apiUrl: 'http://localhost:3000'
```

Comment:
I thought Angular should be on 4200, and MongoDB on 3000?

Reply:
This url is to point to the backend and interact with mongodb. The angular instance needs to point to it to activate the backend functionality.